challenges-webapp/src/main/webapp/vue-app/challenges/components/ChallengeDrawer.vue

```diff
@@ -199,6 +214,11 @@ export default {
       this.disabledSave = true ;
       this.checkEnableSaveChallenge();
     },
+    enableCreateButton() {
+      this.validDescription = true ;
+      this.disabledSave = false ;
+      this.checkEnableSaveChallenge();
```

Comment:
Is it possible to change this check in a computed property ?

Reply:
scope misnamed!
we need to check every time when get data from other components